### PR TITLE
LPS-150040 [Raylife AP] Adjust Sign In Page redirection

### PR DESCRIPTION
**Pull request by Mylena-Rodrigues**

### CODE REVIEW

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/sign-in/index.css [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
 .raylife-login-form li:nth-child(2) {
 	position: absolute;
+	bottom: -30vh;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/sign-in/index.css:41]: Sort these properties, please.

**Review by silasbsi — COMMENTED** [review]
Needs a little adjustment

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**